lib/urbanopt/scenario/default_reports/end_use.rb

```diff
@@ -65,6 +65,26 @@ def to_hash
         return result
 
     end
+    
+    def merge_end_use!(new_end_use)
```

Comment:
@rawadelkontar in Ruby methods that modify the object `self` are typically denoted with an `!`, these are called [bang methods](http://ruby-for-beginners.rubymonstas.org/objects/bangs.html)